### PR TITLE
REQUIRED_USE qt5 vs. qt6 conflicts are terrible and must be avoided

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

**Review by chewi — APPROVED** [review]

**Review by floppym — APPROVED** [review]
Good to go for transmission. If you want to revbump 4.0.6 that would be fine too.

File: profiles/arch/x86/package.use.mask [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2024-09-12)
+# dev-python/PyQt6 and dev-python/pyside6 aren't keyworded here
+dev-python/QtPy pyqt6 pyside6
```
Comment (ionenwks) [inline profiles/arch/x86/package.use.mask:6]: self-note: I'll try to remember to drop pyqt6 here after https://bugs.gentoo.org/935070 (been meant to have x86 for a while, even stable, but been stalled for some reason)

**Review by ionenwks — APPROVED** [review]
Check the qa-report link, it shows green but it actually isn't (again), beside that ok with me once resolved.

**Comment by ionenwks** [comment]
wrt the PyQt6-WebEngine bit in the qa-report link, I filed https://bugs.gentoo.org/939558

**Review by AndrewAmmerlaan — APPROVED** [review]

**Pull request merged**